server/server_test.go

```diff
@@ -317,6 +317,22 @@ func TestCompileV1(t *testing.T) {
 			note: "error: bad unknown",
 			trs:  []tr{{http.MethodPost, "/compile", `{"unknowns": ["input."], "query": "true"}`, 400, ""}},
 		},
+		{
+			note: "double equals rewrite",
```

Comment:
Nit: I'd consider putting this test in the rego package.

Reply:
I was wondering about that. I didn't see many similar tests in there and this seemed like where a lot of that was happening. I'll add some coverage in there too.. good excuse for me to play around with the rego package API's anyway.

Reply:
Moved the test to rego_test.go in latest update